app/components/search_result/no_result/filters_component.rb

```diff
@@ -3,7 +3,9 @@
 module SearchResult
   module NoResult
     class FiltersComponent < Blacklight::ConstraintsComponent
-      def query_constraints; end
+      def render?
+        !(@search_state.filters.blank? && @search_state.clause_params.blank?)
```

Comment:
```suggestion
        @search_state.filters.present? || @search_state.clause_params.present?
```

Maybe that's clearer, or at least takes less reasoning?

Reply:
FWIW I have a slight preference for inverting the logic. I wouldn't immediately recognize that as Blacklight code.